docs/configure/automation/integrations.md

```diff
@@ -71,6 +71,109 @@ if (ActionItem.ResourceNamespace === "api") {
 
 If the Action Item associated with the ticket is marked as `Resolved` or `Fixed`, the third party ticket will automatically close.
 
+### Customizable fields:
+Insights provides ability to customize how the ticket is created at the target ticketing provider in some extent, by adding an a 4th parameter to `createTicket` function.
+This parameter is a generic key/value object. 
+For this customization some level os knowledge about the target Ticket Provider is required.
+
+
+#### Jira example:
+
+For reference, Jira API for creating ticket can be found here:
+https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/
+
+Properties sent in the customizable fields field will overwrite the `fields` parameter in Jira API:
+
+```js
+customizableFields = {
+   "assignee": {
+      "id": "user123"
+   },
+   "customfield_10000": "09/Jun/19",
+   "issuetype": {
+      "name": "Task"
+   }
+}
+if (ActionItem.ResourceNamespace === "api") {
+  createTicket("Jira", "API", ["bug"], customizableFields)
+}
+```
+In the example above the ticket will be assigned to "user123", we are adding a custom field named "customfield_10000" with value "09/Jun/19" and the issue type will be "Task", instead of our default value "Bug".
+
+The payload sent to Jira will be like:
+```js
+fields: {
+   "assignee": {
+      "id": "user123"
+   },
+   "customfield_10000": "09/Jun/19",
+   "issuetype": {
+      "name": "Task"
+   }
+}
+```
+
+#### GitHub example:
+
+Github provides some limited numbers of fields that could be customized:
```

Comment:
Github provides a limited number of fields that can be customized:

Reply:
done